.github/workflows/build-image.yml

```diff
@@ -0,0 +1,56 @@
+name: Build and push image variant
+
+on:
+  workflow_call:
+    inputs:
+      ARCHES:
+        required: true
+        type: string
+      CUDA_VER:
+        required: true
+        type: string
+      LINUX_VER:
+        required: true
+        type: string
+      PYTHON_VER:
+        required: true
+        type: string
+      TAG:
+        required: true
+        type: string
+
+jobs:
+  docker-build:
+    strategy:
+      matrix:
+        ARCH: ${{ fromJSON(inputs.ARCHES) }}
```

Comment:
The consequence of not including any other values in this matrix is that the GHAs sidebar only shows the architecture name (see screenshot below).

This can make it difficult to differentiate the builds.

It might be a good idea to include those other matrix values here.

![image](https://github.com/rapidsai/mambaforge-cuda/assets/7400326/2702988d-2dde-43ef-91c9-cbe7a4f5d968)